src/package.coffee

```diff
@@ -461,6 +459,26 @@ class Package
   activationShouldBeDeferred: ->
     @hasActivationCommands() or @hasActivationHooks()
 
+  PackagesToActivateImmediately = [
+    # core
+    'status-bar',
+    'tabs',
+    'tree-view',
+    'wrap-guide',
```

Comment:
Can we please have `notifications` here as well? It'll fix https://github.com/atom/atom/issues/9522